docs/en/users/10_filter.md

```diff
@@ -84,6 +84,13 @@ You can use the search field to further refine results:
 		* `date:PT30M/` (past thirty minutes)
 		* `date:PT90S/` (past ninety seconds)
 		* `date:P1DT1H/` (past one day and one hour)
+	* From a given time back to the earliest:
+		* `!date:P1M` (dynamically from one month before the current time)  
```

Comment:
That excludes articles from the past one month (shows only articles older than that)